tutorials/primer/.pages

```diff
@@ -0,0 +1,10 @@
+nav:
+  - ...
+  - elements-shards.md
+  - anatomy-function.md
+  - anatomy-shard.md
+  - manipulate-data.md
+  - control-flow.md
+  - appendix-a
+  - appendix-b
+  - appendix-c
```

Comment:
Missing the `.md` extension.

Alternatively, I think the appendices should be in their own section (thus create a folder).

```suggestion
  - appendices
```

When doing so, it is good to keep "Appendix" in the document title, however it is redundant in the URL and in the navigation panel. This can be overridden using the YAML front-matter as such (e.g. for Appendix A):
```md
---
title: A - Developer toolbox
---
# Appendix A - Developer toolbox
```
See https://squidfunk.github.io/mkdocs-material/reference/#usage